test/t8_cmesh/t8_gtest_multiple_attributes.cxx

```diff
@@ -0,0 +1,181 @@
+/*
+  This file is part of t8code.
+  t8code is a C library to manage a collection (a forest) of multiple
+  connected adaptive space-trees of general element classes in parallel.
+
+  Copyright (C) 2015 the developers
+
+  t8code is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+
+  t8code is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with t8code; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+#include <gtest/gtest.h>
+#include <t8_cmesh.h>
+#include <t8_schemes/t8_default/t8_default_cxx.hxx>
+#include "t8_cmesh/t8_cmesh_partition.h"
+#include <t8_cmesh_vtk.h>
+#include <t8_geometry/t8_geometry_implementations/t8_geometry_linear.h>
+
+/* Test if multiple attributes are partitioned correctly. */
+
+/** Construct \a num_trees many cubes each of length 1 connected along the x-axis 
+ * with only one attribute, or with additional attributes.
+*/
```

Comment:
We should move this function int `t8_cmesh/t8_cmesh_examples`

Reply:
Done